python/instrumentation/openinference-instrumentation-litellm/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenInference liteLLM Instrumentation
+
+Python autoinstrumentation library for liteLLM calls
```

Comment:
`Python autoinstrumentation for the LiteLLM package`

Reply:
done